config/extension_import_variables.go

```diff
@@ -61,6 +61,18 @@ func (item *ImportVariables) loadVariablesFromFile(file string) error {
 
 func (item *ImportVariables) loadVariables(newVariables map[string]interface{}, source options.VariableSource) {
 	for _, nested := range item.NestedObjects {
+		for k1, v1 := range newVariables {
+			// Simplify the reference to variables in case the key is repeated (ex: project.project.value can be directly acceeded with project.value)
+			if v1, isMap := v1.(map[string]interface{}); isMap {
```

Comment:
I think variable names could be a bit better
Here v1 is reassigned but we now know that it's a map. It could be
```
for k1, v1 := range newVariables {
	// Simplify the reference to variables in case the key is repeated (ex: project.project.value can be directly acceeded with project.value)
	if map1, isMap := v1.(map[string]interface{}); isMap {
		if map2, isMap := map1[k1].(map[string]interface{}); isMap {
			for k3, v3 := range map2 {
				if _, exist := map1[k3]; !exist {
					map1[k3] = v3
				}
			}
		}
	}
}
```